data/data/gcp/network/firewall-control.tf

```diff
@@ -51,20 +51,19 @@ resource "google_compute_firewall" "master_ingress_from_health_checks" {
 
 resource "google_compute_firewall" "master_ingress_mcs" {
   name    = "${var.cluster_id}-master-in-mcs"
-  network = google_compute_network.cluster_network.self_link
+  network = local.cluster_network
 
   allow {
     protocol = "tcp"
     ports    = ["22623"]
   }
 
-  source_ranges = [var.network_cidr, google_compute_address.master_nat_ip.address, google_compute_address.worker_nat_ip.address]
```

Comment:
what does this change mean?

Reply:
this changes the behavior of installer created VPC setup, we should either open issue to track this change as we cannot modify the existing behavior.

Reply:
I don't think we can determine the master & worker nat ip addresses just given the vpc & subnets. I updated so this doesn't change the behavior for IPI VPCs.